client/components/bloganuary-header/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import BloganuaryIcon from 'calypso/components/blogging-prompt-card/bloganuary-icon';
+import isBloganuary from 'calypso/data/blogging-prompt/is-bloganuary';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import './style.scss';
+
+const BloganuaryHeader = () => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const trackBloganuaryMoreInfoClick = () => {
+		dispatch( recordTracksEvent( 'reader_bloganuary_more_info_click' ) );
```

Comment:
We should probably change the name to fit with the other event standards with a `calypso` prefix. ex. `calypso_reader_bloganuary_banner_more_info_click`